### DISCUSSION TITLE
JS Client "Logging"

### DISCUSSION THREAD

**Discussion by charlie17** [supabase/supabase#1683] [Questions]
I understand from [here](https://github.com/supabase/supabase/discussions/479) that the ability to work with Postgres logs is coming to the Supabase dashboard in the future (big thumbs up).

But in the meantime, is there some way to see exactly what SQL syntax is getting executed as a result of a JS client library request?

**Comment by awalias** [comment]
@burggraf has a tool for this 

https://github.com/burggraf/SupabaseLogQuery

but I *think* that the SQL logs are currently set to error/connection logging only - do you have any other advice her @burggraf ?

**Reply by burggraf** [reply]
LOL - I forgot I wrote this!  I don't have anything to add at this point, but if she has problems with the log query stuff I'll be happy to help out.
